http-webhook.md

```diff
@@ -0,0 +1,339 @@
+# HTTP 1.1 Web Hooks for Event Delivery
+
+## Abstract
+
+"Web hooks" are a popular pattern to deliver notifications between applications
+and via HTTP endpoints. In spite of pattern usage being widespread, there is no
+formal definition for Web Hooks. This specification aims to provide such a
```

Comment:
If it is meant to be generic, why not split it out as its own draft?

Reply:
It is its own draft. Microsoft would like to take this specific spec to IETF.